modules/statics/src/account.ts

```diff
@@ -33,7 +33,6 @@ export class AccountCoin extends BaseCoin {
 
   constructor(options: AccountConstructorOptions) {
     super({
-      isToken: false,
```

Comment:
What does this imply? We have account coins which are tokens too?

Reply:
It doesn't imply anything -- this field was never actually being set because `AccountConstructorOptions` requires the implementors to set `isToken` themselves. You can see that it is set by default to `false` for a normal `AccountCoin`: https://github.com/BitGo/BitGoJS/blob/f9a3a38617998e66fe62da7be655e97c37cd979b/modules/statics/src/account.ts#L159 -- and overridden to `true` for tokens

Reply:
Gotcha, thanks!